src/features/bulk/store/model.ts

```diff
@@ -0,0 +1,29 @@
+import { CustomFile } from '@amsterdam/bmi-component-library';
+
+type IBulkMetadataFieldId = string;
+
+export interface IBulkMetadataFile {
+	id: string;
+	url: string;
+	uploadedFile: CustomFile;
+	metadata?: IBulkMetadataFileMetadata[];
+}
+
+export interface IBulkMetadataFileMetadata {
+	id: IBulkMetadataFieldId;
+	value: string | number | boolean;
+}
+
+export interface IBulkMetadataField {
+	id: IBulkMetadataFieldId;
+	label: string;
+	value: string | number | boolean;
+	changeIndividual: boolean;
+}
+
+export interface IBulkMetadataState {
+	currentStep: 'upload' | 'selectFields' | 'editFile'
```

Comment:
I guess this is fine, but I can imagine that you can put these "steps" in an enum since they all represent the same "type" of input

Reply:
fixed!